prowler/providers/azure/azure_provider.py

```diff
@@ -615,7 +616,7 @@ async def get_azure_identity():
                 logger.critical(
                     "It was not possible to retrieve any subscriptions, please check your permission assignments"
                 )
-                sys.exit(1)
+                raise SystemExit
```

Comment:
Add a TODO here to replace it with our custom exceptions once they are ready.

Reply:
Done